static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -405,6 +411,36 @@ class IssueRuleEditor extends AsyncView<Props, State> {
     this.fetchPreview(true);
   }, 1000);
 
+  // As more incompatible combinations are added, we will need a more generic way to check for incompatibility.
+  checkIncompatibleRule = debounce(() => {
+    const {rule} = this.state;
+    if (
+      !rule ||
+      !this.props.organization.features.includes('issue-alert-incompatible-rules')
+    ) {
+      return;
+    }
+
+    const conditions = rule.conditions;
+    let firstSeen = 0;
+    let regression = 0;
+    let reappearance = 0;
+    for (let i = 0; i < conditions.length; i++) {
+      const id = conditions[i].id;
+      if (id.endsWith('FirstSeenEventCondition')) {
+        firstSeen = 1;
+      } else if (id.endsWith('RegressionEventCondition')) {
+        regression = 1;
+      } else if (id.endsWith('ReappearanceEventCondition')) {
+        reappearance = 1;
+      }
```

Comment:
change this to a one liner with a tuple
https://docs.python.org/3/library/stdtypes.html#str.endswith

`.endwith(('FirstSeenEC', 'RegressionEC', 'ReappearEC'))`